package.json

```diff
@@ -90,28 +104,26 @@
     "babel-core": "6.26.0",
     "babel-jest": "22.1.0",
     "babel-preset-env": "1.6.1",
-    "bundlesize": "0.15.3",
```

Comment:
@rosskevin why are we removing bundlesize?

Reply:
We aren't creating a UMD bundle anymore, not sure if we can size a non-compressed folder of files.  I'll check.

Reply:
Added back to test rollup-produced CJS file